prepare/kotlin-compiler-internal-test-framework/build.gradle.kts

```diff
@@ -1,21 +1,54 @@
+import org.gradle.jvm.tasks.Jar
+
 plugins {
     java
+    `maven-publish`
 }
 
 val embedded by configurations
 
 dependencies {
+    embedded(project(":compiler:serialization")){ isTransitive = false }
+    embedded(projectTests(":compiler:tests-common-jvm6")){ isTransitive = false }
+    embedded(project(":kotlin-scripting-compiler-impl")){ isTransitive = false }
+    embedded(projectTests(":compiler:test-infrastructure-utils")){ isTransitive = false }
+    embedded(projectTests(":compiler:tests-compiler-utils")){ isTransitive = false }
```

Comment:
I suggest you to take a look at `:compiler:tests-common-new` module, which contains new compiler tests framework. We in compiler teams actively migrate to it from `:compiler:tests-common`

Reply:
Thanks for pointing me to it! I have updated our tests to use this framework instead, but it required a few more changes